src/applications/gi-sandbox/containers/GiBillApp.jsx

```diff
@@ -66,13 +67,15 @@ export function GiBillApp({
           {constants.error && <ServiceError />}
           {!(constants.error || constants.inProgress) && (
             <DowntimeNotification appTitle={'GI Bill Comparison Tool'}>
-              <div className="tool-description">
-                <h1>GI Bill® Comparison Tool</h1>
-                <p className="vads-u-font-size--h3 vads-u-color--gray-dark">
-                  Use the GI Bill Comparison Tool to see how VA education
-                  benefits can pay for your education.
-                </p>
-              </div>
+              {Object.keys(profile.attributes).length === 0 && (
```

Comment:
put `const onProfilePage = location.pathname.includes('/profile');`
```suggestion
              {!onProfilePage && (
```

Reply:
`const location = useLocation();`
and 
`import { useLocation } from 'react-router-dom';`